src/containers/ChildcareFormOptions.js

```diff
@@ -126,7 +126,7 @@ function ChildcareFormDetails({ id, onSave, request }) {
         <Text
           as="div"
           type={TEXT_TYPE.NOTE}
-          css={{ color: Color.GRAY_75 }}
+          css={{ color: Color.GRAY_75, marginBottom: Space.S30 }}
```

Comment:
could we put container styles at the top of the file? less refactor for later on :)